changelog/Microsoft.AAD.Reporting.json

```
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "a158aac3-4b31-4dc8-8449-cbf1898f7a8b",
+          "ApiChange": "Property",
+          "ChangedApiName": "lastUpdatedDateTime",
+          "ChangeType": "Addition",
+          "Description": "Added the **lastUpdatedDateTime** property to the [userRegistrationDetails](https://learn.microsoft.com/en-us/graph/api/resources/userRegistrationDetails?view=graph-rest-beta) resource.",
+          "Target": "userRegistrationDetails"
+        }
+      ],
+      "Id": "a158aac3-4b31-4dc8-8449-cbf1898f7a8b",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2023-04-25T16:06:45.9062048Z",
+      "WorkloadArea": "Reports and audit",
```

Should match the top-level name on the TOC
```suggestion
      "WorkloadArea": "Reports",
```